src/inc/msquic.h

```diff
@@ -663,7 +664,8 @@ typedef struct QUIC_SETTINGS {
             uint64_t GreaseQuicBitEnabled                   : 1;
             uint64_t EcnEnabled                             : 1;
             uint64_t HyStartEnabled                         : 1;
-            uint64_t RESERVED                               : 29;
+            uint64_t EncryptionOffloadEnabled               : 1;
```

Comment:
Let's call this `EncryptionOffloadAllowed` instead of enabled. It can only truely be _enabled_ if the HW & platform supports it.